spek-extension/robolectric/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply from: "$rootDir/gradle/common/dependencies.gradle"
+apply from: "$rootDir/gradle/common/kotlin-jvm.gradle"
+apply from: "$rootDir/gradle/common/junit-jupiter.gradle"
+apply plugin: 'org.jetbrains.dokka'
+apply plugin: 'java-library'
+
+dependencies {
+    compile project(':spek-dsl:jvm')
```

Comment:
Since you are using `java-library`, can you change the `compile` to `implementation`?